### ISSUE TITLE
Unable to place FnO orders for BankNifty. able to place the orders in NSE but not NFO

### ISSUE THREAD

**Issue by pnandan13a** [jerokpradeep/pya3#16]
print(alice.place_order(transaction_type = TransactionType.Sell,instrument = alice.get_instrument_by_token('NFO',55873),quantity = 25,order_type = OrderType.Market,product_type = ProductType.Delivery,price = None,trigger_price = None,stop_loss = None,square_off = None,trailing_sl = None,is_amo = False))

response

[{'emsg': 'Invalid Order Type or Product Code', 'stat': 'Not_Ok'}]

**Comment by pnandan13a** [comment]
only Intraday works **but Normal Orders fails**

print(alice.place_order(transaction_type = TransactionType.Buy,instrument = alice.get_instrument_by_token('NFO',55873),quantity = 25,order_type = OrderType.Market,product_type = ProductType.Intraday,price = None,trigger_price = None,stop_loss = None,square_off = None,trailing_sl = None,is_amo = False))

[{'stat': 'Ok', 'NOrdNo': '220801000171142'}]

**Comment by Vignesh-Devoloper** [comment]
Hi @pnandan13a,
You can not able take the sell orders in delivery. You can place only buy orders in delivery.

**Comment by pnandan13a** [comment]
Hi @Vignesh-Devoloper ,
           even buy orders fails with error [{'emsg': 'Invalid Order Type or Product Code', 'stat': 'Not_Ok'}]
I am able to place  delivery\normal orders  manually from app\web
     Can you please have  a look.

**Comment by Vignesh-Devoloper** [comment]
Yes, we find the issue @pnandan13a. Fixed the bug in Version 1.0.13. Please update the pya3 library and try the same request.

**Comment by pnandan13a** [comment]
thanks for your support, will update library Version 1.0.13 and keep you posted

**Comment by pnandan13a** [comment]
When will version 1.0.13 be available?
